### PR TITLE
Fix Exercise Answer Feedback for use in Exercises

### PR DESCRIPTION
**Pull request by jivewise**
- [x] Fixes answer feedback for exercises, see screenshot below
- [x] Add in specs for answers and answer feedback

screen shot 2016-02-25 at 3 37 09 am

### CODE REVIEW

**Comment by philschatz** [comment]
any thoughts @openstax/tutor-fe ?

File: src/components/question/answer.cjsx [hunk 1]
```diff
@@ -117,7 +118,7 @@ Answer = React.createClass
       </div>
 
     if @props.show_all_feedback and answer.feedback_html
-      feedback = <Feedback key='question-mc-feedback'>{answer.feedback_html}</Feedback>
+      feedback = <Feedback key='question-mc-feedback' children={answer.feedback_html} />
```
Comment (nathanstitt) [inline src/components/question/answer.cjsx:122]: Curious, but why'd you set the children as a prop here?  The two methods are equivalent aren't they?

Comment (jivewise) [inline src/components/question/answer.cjsx:122]: Ah, yea, I just updated that when I was debugging before i realized what was going on with the Feedback object.

Comment (nathanstitt) [inline src/components/question/answer.cjsx:122]: heh, I thought maybe you'd discovered something new about how it renders `children` :)

Comment (jivewise) [inline src/components/question/answer.cjsx:122]: Nope, I was getting a weird type error, and I thought it meant `propType`, so I just put it as an attribute to make sure that wasn't it.

Comment (philschatz) [inline src/components/question/answer.cjsx:122]: @jivewise can you make a new PR which cleans up the syntax?

**Comment by jivewise** [comment]
Yep!

**Pull request merged**